test/integration/websocket_integration_test.cc

```diff
@@ -31,19 +31,12 @@ Http::TestRequestHeaderMapImpl upgradeRequestHeaders(const char* upgrade_type =
 }
 
 Http::TestResponseHeaderMapImpl upgradeResponseHeaders(const char* upgrade_type = "websocket") {
-  return Http::TestResponseHeaderMapImpl{{":status", "101"},
-                                         {"connection", "upgrade"},
-                                         {"upgrade", upgrade_type},
-                                         {"content-length", "0"}};
+  return Http::TestResponseHeaderMapImpl{
+      {":status", "101"}, {"connection", "upgrade"}, {"upgrade", upgrade_type}};
```

Comment:
As long as you're in here, I think this should be a 200.

Reply:
For http/1.1, it seems like it has to be 101. But maybe something different happens in h2?

From https://tools.ietf.org/html/rfc6455#section-4.2.2:
>  5.  If the server chooses to accept the incoming connection, it MUST
>       reply with a valid HTTP response indicating the following.
>
>       1.  A Status-Line with a 101 response code as per RFC 2616
>           [RFC2616].  Such a response could look like "HTTP/1.1 101
>           Switching Protocols".